go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/Shopify/sarama v1.38.1
 	github.com/alecthomas/kingpin/v2 v2.3.2
 	github.com/alecthomas/units v0.0.0-20211218093645-b94a6e3cc137
-	github.com/aws/aws-sdk-go v1.44.304
+	github.com/aws/aws-sdk-go v1.44.311
```

Comment:
Do we need to be updating any of our dependencies in this PR? I'm particularly worried about the blast radius of modifying pkg/agentproto.

Reply:
I had to update the version of these 3 dependencies:
1. github.com/grafana/loki
2. github.com/grafana/loki/pkg/push - to have the new request proto model
3. github.com/grafana/windows_exporter - the previous version did not exist in the repo, so, I failed to run `go get ....` or `go mod tidy` . 

and these updates bring some other dependencies updates.